driver/pace/driver/report.py

```diff
@@ -6,7 +6,7 @@
 
 import numpy as np
 
-from pace.util.mpi import MPI
+from .comm import Comm
```

Comment:
Should this be `from pace.util.comm import Comm`?

Reply:
Yes, updated.